templates/mcq.jsx

```diff
@@ -0,0 +1,104 @@
+import Adapt from 'core/js/adapt';
+import { templates, classes, html, compile } from 'core/js/reactHelpers';
+
+export default function(model, view) {
+  const data = model.toJSON();
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
```

Comment:
Should we be following @oliverfoster's code here and [adding the globals to `data`](https://github.com/adaptlearning/adapt-contrib-accordion/commit/3690efcdce4009b11f79c6a3bc53c5115c648eca#diff-a08bbbbf19062efb2756675dd9a7a1dfR6)?

Reply:
Probably. @oliverfoster What do you think?

Reply:
If you've got any `compile` directives the handlebars templates will be expecting it although I doubt it was ever used for anything but _globals._learnerInfo etc